packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx

```diff
@@ -976,7 +976,13 @@ const SavedChartsHeader: FC<SavedChartsHeaderProps> = ({
                             }
                         });
 
-                        history.push('/');
+                        if (search) {
```

Comment:
Should we check `dashboardUuid` instead of `search` here ? 

For example, if the chart has another search (fromSpace), and I delete a chart, it throws an error because it takes me to `projects/3675b69e-8324-4110-bdca-059031aa8da3/dashboards/null` 

[Screencast from 06-08-24 11:15:36.webm](https://github.com/user-attachments/assets/ce1654aa-6661-4a77-b884-e11d94a849d3)

Reply:
yes that's correct. We should check `dashboardUuid` because it redirecting to this `dashboardUuid` let me correct this.